tools/auth_keys_generator.go

```diff
@@ -71,7 +74,7 @@ func fileExists(filename string) bool {
 }
 
 func newFile(filename string, data []byte, perm os.FileMode) error {
-	f, err := os.OpenFile(filename, os.O_WRONLY|os.O_CREATE|os.O_EXCL, perm)
+	f, err := os.OpenFile(filename, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, perm)
```

Comment:
please explain why.
Isn't `O_EXCL` what we want? 
```golang
O_EXCL   int = syscall.O_EXCL   // used with O_CREATE, file must not exist.
```
([source](https://golang.org/pkg/os/#pkg-constants))

Reply:
Wanted to support updating the files as we added some new tokens (auth-tokenAdminString/auth-tokenUnallowedString). So if a user already generated keys, instead of failing it would update the tokens and create the new ones. Do you think it's better to fail and ask the user to run 'make clean' in such case?

Reply:
IMO, files override should be deliberate.

This currently prevents a case in which a tester loads a local test cert to assisted-service, but later on, overrides the token they need to use in order to query the API.

This is why I wrote it to fail like that.
Nevertheless, a `"--force"` flag can be added to this keys generator so we'll know the regeneration is done by purpose.

Reply:
ok, removed the override support. users should simply invoke clean manually to remove stale tokens.